crates/holochain/src/test_utils.rs

```diff
@@ -126,19 +126,42 @@ pub struct TestNetwork {
     network: Option<HolochainP2pRef>,
     respond_task: Option<tokio::task::JoinHandle<()>>,
     dna_network: HolochainP2pDna,
+
+    /// check op data calls
```

Comment:
What would be a good doc for this? I can't tell what this is for by looking at it.